pkg/distributor/distributor.go

```diff
@@ -757,22 +757,21 @@ func (d *Distributor) validateSeries(nowt time.Time, ts *mimirpb.PreallocTimeser
 	for i := 0; i < len(ts.Exemplars); {
 		e := ts.Exemplars[i]
 		if err := validateExemplar(d.exemplarValidationMetrics, userID, ts.Labels, e); err != nil {
-			// An exemplar validation error prevents ingesting samples
```

Comment:
how about just 
// ignore invalid exemplars
if (validateExemplar(d.exemplarValidationMetrics, userID, ts.Labels, e) != nil || !validateExemplarTimestamp(d.exemplarValidationMetrics, userID, minExemplarTS, maxExemplarTS, e)) {
    ts.DeleteExemplarByMovingLast(i)
    continue
}

Reply:
I think the drawback with this simplification is losing the distinction between ignoring exemplars for being invalid and for having an out-of-range timestamp. The first case is documented via a comment: `// OTel sends empty exemplars by default, so let's just skip invalid ones.`.